components/Payment/installation.rst

```diff
@@ -1,18 +1,11 @@
 Installation
 ============
 
-We assume you're familiar with `Composer <http://packagist.org>`_, a dependency manager for PHP.
-Use following command to add the component to your `composer.json` and download package.
+You can install the component in 2 different ways:
```

Comment:
Think the main reason was that all the information needed to install components (along with a link to composer and examples of use) are already located on 'Using Components' page, so specifying only the package name / git repository on a component page should be enough. IMO